src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1297,4 +1297,46 @@ oben genannten Länder. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome m
 
 "AutomaticSharingConsent_DataProcessingDetailInfo" = "Ausführliche Informationen zur Datenverarbeitung und Ihrem Einverständnis.";
 
+/* Thank you screen */
+"Thank_You_Title" = "Vielen Dank!";
 
+"Thank_You_SubTitle" = "Vielen Dank, dass Sie mithelfen, die Verbreitung von Corona zu bekämpfen!";
```

Comment:
"Verbreitung des Corona-Virus" als Vorschlag?

Reply:
Thanks for your proposal. Changes like this may have a wider impact, so we'll keep it in mind and will double check, but would keep "Verbreitung von Corona" for now.